File: app/src/main/java/com/fractalwrench/crazycats/mocks/MockImageListCellDelegate.java
package com.fractalwrench.crazycats.mocks;


import com.fractalwrench.crazycats.image.data.ImageData;
import com.fractalwrench.crazycats.image.list.ImageListView;

public class MockImageListCellDelegate implements ImageListView.CellDelegate {

    public ImageData ImageData;

    @Override
    public void onImageCellClicked(ImageData ImageData) {
        this.ImageData = ImageData;
    }

}

File: app/src/main/java/com/fractalwrench/crazycats/mocks/MockImageListCellDelegate.kt
package com.fractalwrench.crazycats.mocks


import com.fractalwrench.crazycats.image.data.ImageData
import com.fractalwrench.crazycats.image.list.ImageListView

class MockImageListCellDelegate : ImageListView.CellDelegate {

    var imageData: ImageData? = null

    override fun onImageCellClicked(imageData: ImageData) {
        this.imageData = imageData
    }

}
